libobs-d3d11/d3d11-subsystem.cpp

```diff
@@ -656,6 +656,21 @@ static bool FastClearSupported(UINT vendorId, uint64_t version)
 	return aa >= 31 && bb >= 0 && ccccc >= 15 && ddddd >= 2737;
 }
 
+static bool HAGSPriorityFixed(UINT vendorId, uint64_t version)
+{
+	/* Always true for non-NVIDIA GPUs */
+	if (vendorId != 0x10de)
+		return true;
+
+	const uint16_t aa = (version >> 48) & 0xffff;
+	const uint16_t bb = (version >> 32) & 0xffff;
+	const uint16_t ccccc = (version >> 16) & 0xffff;
+	const uint16_t ddddd = version & 0xffff;
+
+	/* Check for NVIDIA driver version >= 31.0.15.3699 */
+	return aa >= 31 && bb >= 0 && ccccc >= 15 && ddddd >= 3699;
```

Comment:
Unfortunately this driver version does not actually seem to fix all of it yet, gotta wait for verification.